deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
Was 1264e09 the tip of the branch you used to recreate the worker? That one didn't have `setuid mmw` in the upstart job, so it would have run the service as root there. I just tried running that branch with `setuid mmw` added to upstart and got this error:

```sh
...

==> worker: Checking for guest additions in VM...                                                                                                                            
==> worker: Setting hostname...                                                                                                                                              
==> worker: Configuring and enabling network interfaces...                                                                                                                   
==> worker: Mounting shared folders...                                                                                                                                       
    worker: /aws => /Users/azavea/.aws                                                                                                                                       
    worker: /opt/app => /Users/azavea/dvlp/model-my-watershed/src/mmw                                                                                                        
    worker: /vagrant => /Users/azavea/dvlp/model-my-watershed                                                                                                                
    worker: /opt/rwd-data => /private/tmp                                                                                                                                    
==> worker: Running provisioner: ansible...                                                                                                                                  
    worker: Running ansible-playbook...         

PLAY [workers] *****************************************************************                                                                                             
                                                                                                                                                                             
TASK [setup] *******************************************************************                                                                                             
Friday 04 August 2017  16:23:48 -0400 (0:00:00.297)       0:00:00.297 *********                                                                                              
ok: [worker]                                                                                                                                                                 
                                                                                                                                                                             
TASK [Update APT cache] ********************************************************                                                                                             
Friday 04 August 2017  16:23:51 -0400 (0:00:02.571)       0:00:02.869 *********                                                                                              
ok: [worker]                                                                                                                                                                 
                                                                                                                                                                             
TASK [azavea.java : Add OpenJDK PPA] *******************************************                                                                                             
Friday 04 August 2017  16:24:08 -0400 (0:00:17.145)       0:00:20.014 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.java : Install OpenJDK JRE (headless)] ****************************                                                                                             
Friday 04 August 2017  16:24:17 -0400 (0:00:08.662)       0:00:28.677 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.java : Install OpenJDK (headless)] ********************************                                                                                             
Friday 04 August 2017  16:25:13 -0400 (0:00:56.317)       0:01:24.995 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.java : Install OpenJDK JRE] ***************************************                                                                                             
Friday 04 August 2017  16:25:31 -0400 (0:00:18.542)       0:01:43.537 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.java : Install OpenJDK] *******************************************                                                                                             
Friday 04 August 2017  16:25:46 -0400 (0:00:14.277)       0:01:57.815 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.java : Determine if 64bit architecture] ***************************                                                                                             
Friday 04 August 2017  16:25:55 -0400 (0:00:09.479)       0:02:07.295 *********                                                                                              
ok: [worker]                                                                                                                                                                 
                                                                                                                                                                             
TASK [azavea.java : Set OpenJDK as the default] ********************************                                                                                             
Friday 04 August 2017  16:25:55 -0400 (0:00:00.099)       0:02:07.394 *********                                                                                              
ok: [worker]                                                                                                                                                                 
                                                                                                                                                                             
TASK [azavea.java : Configure the WebUpd8 repository] **************************                                                                                             
Friday 04 August 2017  16:25:56 -0400 (0:00:00.342)       0:02:07.737 *********                                                                                              
skipping: [worker]  

TASK [azavea.java : Accept Oracle license] *************************************                                                                                             
Friday 04 August 2017  16:25:56 -0400 (0:00:00.077)       0:02:07.814 *********                                                                                              
skipping: [worker]                                                                                                                                                           
                                                                                                                                                                             
TASK [azavea.java : Install Oracle Java] ***************************************                                                                                             
Friday 04 August 2017  16:25:56 -0400 (0:00:00.074)       0:02:07.888 *********                                                                                              
skipping: [worker]                                                                                                                                                           
                                                                                                                                                                             
TASK [azavea.java : Set Oracle Java as the default] ****************************                                                                                             
Friday 04 August 2017  16:25:56 -0400 (0:00:00.077)       0:02:07.966 *********                                                                                              
skipping: [worker]                                                                                                                                                           
                                                                                                                                                                             
TASK [azavea.java : Write ld.so.conf for Java] *********************************                                                                                             
Friday 04 August 2017  16:25:56 -0400 (0:00:00.074)       0:02:08.041 *********                                                                                              
skipping: [worker]                                                                                                                                                           
                                                                                                                                                                             
TASK [azavea.nginx : Configure the Nginx PPA] **********************************                                                                                             
Friday 04 August 2017  16:25:56 -0400 (0:00:00.077)       0:02:08.118 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.nginx : Install Nginx] ********************************************                                                                                             
Friday 04 August 2017  16:26:04 -0400 (0:00:08.054)       0:02:16.173 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.nginx : Delete default site] **************************************                                                                                             
Friday 04 August 2017  16:26:15 -0400 (0:00:10.646)       0:02:26.819 *********                                                                                              
changed: [worker]                                                                                                                                                            
                                                                                                                                                                             
TASK [azavea.nginx : Delete default web root] **********************************                                                                                             
Friday 04 August 2017  16:26:15 -0400 (0:00:00.390)       0:02:27.210 *********                                                                                              
changed: [worker]   

TASK [azavea.nginx : Check Nginx Upstart service definition exists] ************           
Friday 04 August 2017  16:26:15 -0400 (0:00:00.284)       0:02:27.494 *********            
ok: [worker]                                                                               
                                                                                           
TASK [azavea.nginx : Configure Nginx log rotation] *****************************           
Friday 04 August 2017  16:26:16 -0400 (0:00:00.384)       0:02:27.879 *********            
changed: [worker]                                                                          
                                                                                           
TASK [azavea.nginx : Configure Nginx] ******************************************           
Friday 04 August 2017  16:26:16 -0400 (0:00:00.638)       0:02:28.518 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.nginx : Configure Nginx log format] *******************           
Friday 04 August 2017  16:26:17 -0400 (0:00:00.471)       0:02:28.989 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.geoprocessing : Create geoprocessing directory] *******           
Friday 04 August 2017  16:26:17 -0400 (0:00:00.461)       0:02:29.451 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.geoprocessing : Download mmw-geoprocessing JAR] *******           
Friday 04 August 2017  16:26:18 -0400 (0:00:00.280)       0:02:29.731 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.geoprocessing : Configure Nginx site] *****************           
Friday 04 August 2017  16:26:25 -0400 (0:00:07.443)       0:02:37.175 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.geoprocessing : Enable Nginx site] ********************           
Friday 04 August 2017  16:26:25 -0400 (0:00:00.437)       0:02:37.613 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.geoprocessing : Configure geoprocessing service definition] ***   
Friday 04 August 2017  16:26:26 -0400 (0:00:00.232)       0:02:37.845 *********            
changed: [worker]                                                                          
                                                                                           
TASK [model-my-watershed.geoprocessing : Ensure geoprocessing service is running] ***      
Friday 04 August 2017  16:26:26 -0400 (0:00:00.642)       0:02:38.487 *********            
fatal: [worker]: FAILED! => {"changed": false, "failed": true, "msg": "start: Job failed to start\n"}
```